packages/react/src/utils/useLatestRef.ts

```diff
@@ -1,2 +1,27 @@
 'use client';
-export { useLatestRef } from '@floating-ui/react/utils';
```

Comment:
For reference:

[Source link](https://github.com/floating-ui/floating-ui/blob/ae1329d3173f8fd57c3a8808c6d1ae6a5e6d7825/packages/react/src/utils/hooks.ts#L2-L13)

```typescript
import useModernLayoutEffect from 'use-isomorphic-layout-effect';
import {SafeReact} from './safeReact';

export {useModernLayoutEffect};

export function useLatestRef<T>(value: T) {
  const ref = React.useRef<T>(value);
  useModernLayoutEffect(() => {
    ref.current = value;
  });
  return ref;
}
```